### ISSUE TITLE
Image not loading

### ISSUE THREAD

**Issue by adrian-linca** [square/picasso#1284]
The image below is not loading.
Seems to be the same issue that is workaround with isWebPFile in BitmapHunter.
It's reproducible on android 4.2.2 and below.
http://static.cinemagia.ro/img/resize/db/movie/01/76/03/medium-464288l-120x170-b-b11b0234.bmp

**Comment by adrian-linca** [comment]
After more tests it seems that bitmap images (.bmp) are not loaded in general.

**Comment by iamshajeer** [comment]
did you try this link ?
http://stackoverflow.com/questions/24180805/onbitmaploaded-of-target-object-not-called-on-first-load#answers

**Comment by JakeWharton** [comment]
So you're saying BMP formats cannot be streamed but have to be buffered?

**Comment by adrian-linca** [comment]
Yes, buffering was the only way to load the image, not a Picasso issue though, probably an android framework issue.
It only reproduces on android 4.2.2 and below.

**Comment by JakeWharton** [comment]
Do you have a link to an issue about it? I don't mind adding the fix
without it, but this is the first I'm hearing about it.

**Comment by adrian-linca** [comment]
Sorry I don't have a link.
This is something I debugged when I encountered the issue, with the image I added in the first comment.
In that app we rarely had to deal with BMP files so we did not do any fix.
I assume not many use BMP files in mobile apps, maybe that's why there's little info on this.

**Comment by JakeWharton** [comment]
Yes, I suspect few people have run into it as a result. That being said,
the fix is easy. I'll send a PR.
